File: fibonacci.c
#include<stdio.h>

int main(void)
{
    long long a=0,b=1,c;
    int n,i;
    printf("fibonacci\n"); //Headding
    printf("Enter the value of integer n  : ");
    scanf("%d",&n);
    
    for(i=0;i<n;i++)
    {
        printf("%d ",a);
        c=a+b;
        a=b;
        b=c;
        }

    return 0;
}

File: Fibonacci.java
import java.util.*;
class Fibonacci{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter N: "); int n=sc.nextInt();
        long a=0,b=1;
        for(int i=0;i<n;i++){
            System.out.print(a+" ");
            long c=a+b; a=b; b=c;
        }
    }
}
